app/assets/javascripts/discourse/app/modifiers/close-on-click-outside.js

```diff
@@ -0,0 +1,55 @@
+import { registerDestructor } from "@ember/destroyable";
+import Modifier from "ember-modifier";
+import { bind } from "discourse-common/utils/decorators";
+
+export default class CloseOnClickOutside extends Modifier {
+  constructor(owner, args) {
+    super(owner, args);
+    registerDestructor(this, (instance) => instance.cleanup());
+  }
+
+  modify(
+    element,
+    [closeFn, { targetHTML, targetSelector, secondaryTargetSelector }]
```

Comment:
Not sure about the naming of `targetHTML`. It feels like it's asking for an HTML string, but actually it's supposed to be a reference to a DOM element? Maybe something like `targetElement` would be better?

(or... given that it doesn't seem to be used anywhere... do we just drop it?)

Reply:
dropped